src/NzbDrone.Core/DecisionEngine/DownloadDecisionMaker.cs

```diff
@@ -156,6 +156,26 @@ private IEnumerable<DownloadDecision> GetDecisions(List<ReleaseInfo> reports, Se
 
                 if (decision != null)
                 {
+                    var source = pushedRelease ? ReleaseSourceType.ReleasePush : ReleaseSourceType.Rss;
+
+                    if (searchCriteria != null)
+                    {
+                        if (searchCriteria.InteractiveSearch)
```

Comment:
Would a case be cleaner than the nested ifs?

Reply:
In this case it's not possible because these are different properties on `searchCriteria`, not a single property that can be used in a switch statement.